protos/browser_specific_pass_rate_summary.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wptdashboard;
+
+import "bq_table_name.proto";
+import "browser.proto";
+import "engine.proto";
+import "google/protobuf/timestamp.proto";
+import "operating_system.proto";
+
+// The pass rate from a collection of TestResults from the same TestRun that
+// share "base_name" prefix in their TestResult.test_name.
+message BrowserSpecificPassRateSummary {
+  option (gen_bq_schema.table_name) = "browser_specific_pass_rate_summaries";
+
+  OperatingSystem os = 1;
+  string os_version_str = 2;
+
+  Engine engine = 3;
+  Browser browser = 4;
+  string browser_version_str = 5;
```

Comment:
I'd probably aggregate these into 'platform', which can have a _large_ number of entries, and shard into these 3 enums as metadata (if/when needed)

Reply:
I moved `Engine` to be `Browser` metadata, but I think we may want to query against `Browser` and `OperatingSystem` separately sometimes.